srcpkgs/linux4.20/files/x86_64-dotconfig

```diff
@@ -5371,7 +5371,9 @@ CONFIG_DRM_I915=m
 CONFIG_DRM_I915_CAPTURE_ERROR=y
 CONFIG_DRM_I915_COMPRESS_ERROR=y
 CONFIG_DRM_I915_USERPTR=y
-# CONFIG_DRM_I915_GVT is not set
+CONFIG_DRM_I915_GVT=m
+CONFIG_DRM_I915_GVT_KVMGT=m
+CONFIG_DRM_I915_GVT_XENGT=m
```

Comment:
The current kernel does not know about this option, maybe it was removed or merged with some other option. It is completely ignored in the build process and won't end up in the final configuration so there's no point in adding it.

Reply:
I just tried to compile, and it seems to me like it's definitely an option... It actually specifically prompts during the build process if I want to enable the KVMGT option. I may be mistaken though, I lack a lot of experience in this area. https://cateee.net/lkddb/web-lkddb/DRM_I915_GVT.html also shows a similar story